sdk/eventgrid/azure-eventgrid/samples/consume_samples/consume_eg_storage_blob_created_data_sample.py

```diff
@@ -13,7 +13,6 @@
 """
 import json
 from azure.eventgrid import EventGridDeserializer, EventGridEvent
-from azure.eventgrid.systemevents import StorageBlobCreatedEventData
```

Comment:
because the `_deserialize_data` method will be removed from `EventMixin`, it might be useful to add samples (or add within this sample) of how `SystemEventMappings` can be used with the data/type after deserializing cloud/eg events. (if you already have something like this in the works, ignore this comment :) )

Reply:
same here - the samples will significantly change in the above issue i mentioned - so will update there